server/etc/httpd/conf.d/pulp.conf

```diff
@@ -51,7 +53,13 @@ WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp applic
 </Files>
 
 <VirtualHost *:80>
-    IncludeOptional /etc/pulp/vhosts80/*.conf
+    <IfVersion >= 2.4>
+        # Starting in 2.4 if there are 0 matches this is a configuration error
```

Comment:
I read this as describing the behavior prior to 2.4. Before ``IncludeOptional`` existed, we could only use ``Include``, which considers 0 matches to be an error.

For this block, my understanding is that 0 matches would not be an error.

One way or another, I think it would be helpful to clarify this comment.